front/src/setupProxy.js

```diff
@@ -0,0 +1,13 @@
+
+
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function(app) {
+  app.use(
+    '/api',
```

Comment:
This should be `/api/**` according to https://stackoverflow.com/questions/62048348/proxy-all-requests-to-server-in-react-using-createproxymiddleware